k8s/src/main/scala/io/buoyant/k8s/Api.scala

```diff
@@ -161,8 +161,26 @@ case class ObjectReference(
  * create actual subclasses (i.e. FooWatch extends Watch[Foo]) rather than using Watch[Foo]
  * directly, and set the correct Jackson annotations on those, to ensure correct parsing.
  */
-trait Watch[O <: KubeObject] {
+trait Watch[O <: KubeObject] extends Ordered[Watch[O]] {
   def resourceVersion: Option[String]
+
+  private[Watch] def versionNum: Option[Long] = for {
```

Comment:
what is the visibility scope here?  Is this just equivalent to `private`?

Reply:
yeah, i just thought that – since Scala's visibility modifiers can be confusing – it was clearer to make the scope explicit?